### PR TITLE
DPC-264: Patient/$everything

### PR DESCRIPTION
**Pull request by embh**
**Why**

Vendors need a way to synchronously retrieve all data about a patient.

**What Changed**

New `/Patient/{patient ID}/$everything` endpoint returns all `Coverage`, `ExplanationOfBenefit`, and `Patient` resources for a given patient.

**Tickets closed**:

[DPC-264](https://jiraent.cms.gov/browse/DPC-264)

**Checklist**

- [x] All tests are passing via `make ci-app` (app change) and `make ci-web` (website change)
- [x] Swagger documentation has been updated
- [x] FHIR documentation has been updated
- [x] Any required dpc-ops changes have a PR submitted and mentioned in this ticket
- [x] Any manual migration steps are documented, scripts written (where applicable), and tested
- [x] Before merging, any required dpc-ops changes have been approved and merged into master of the dpc-ops repo

### CODE REVIEW

File: dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/PatientResource.java [hunk 1]
```diff
@@ -145,6 +156,56 @@ public Patient getPatient(@ApiParam(value = "Patient resource ID", required = tr
...
+                .execute();
+
+        if (provider == null) {
+            throw new WebApplicationException(HttpStatus.UNAUTHORIZED_401);
+        }
+
+        final Patient patient = getPatient(patientId);
+        final String patientMbi = FHIRExtractors.getPatientMBI(patient);
+
+        if (!isPatientInRoster(patientId, FHIRExtractors.getProviderNPI(provider), organization.getID())) {
```
Comment (whuang85) [inline dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/PatientResource.java:195]: With the look back now in the aggregation engine, I do check if the patient is within the roster of the provider/org, so I think this check is just a duplicate check maybe?

Comment (embh) [reply dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/PatientResource.java:196]: If I take this out, it has to wait for the lookback work to be merged, right?

Comment (dhgreene) [reply dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/PatientResource.java:196]: I'm more worried about missing a check now than about optimizing how many times we look in the roster.  This is not an expensive operation, correct?  Shall we leave it in?

**Comment by embh** [comment]
Closing stale PR; will create new one for this work.

**Pull request closed without merging**